### PR TITLE
Fix FAB margin on pre-lollipop

### PR DESCRIPTION
**Pull request by ogapants**
## Issue
- #220 

## Overview (Required)
- Remove `styles_compat.xml` (I will return if you plan to use it in the future)
- and add `fab_margin` to `dimens.xml`

## Links
none

## Screenshot
Before | After
:--: | :--:
 |

### CODE REVIEW

File: app/src/main/res/layout/fragment_session_detail.xml [hunk 1]
```diff
@@ -292,10 +292,10 @@
 
         <android.support.design.widget.FloatingActionButton
                 android:id="@+id/fab"
-                style="@style/FabMargin"
                 android:layout_width="wrap_content"
                 android:layout_height="wrap_content"
-                android:layout_gravity="bottom|right|end"
+                android:layout_gravity="bottom|end"
+                android:layout_margin="@dimen/fab_margin"
```
Comment (konifar) [inline app/src/main/res/layout/fragment_session_detail.xml]: `@dimen/space_16dp` is enough!

**Comment by konifar** [comment]
Thanks! Commented 👍

File: app/src/main/res/layout/fragment_session_detail.xml (hunk 1 above)
Comment (ogapants) [inline app/src/main/res/layout/fragment_session_detail.xml]: Oops, okey!

**Comment by konifar** [comment]
Perfect 💯

**Pull request merged**